### PR TITLE
Add sigmoid activation to Paddle frontend and test

### LINKED ISSUE #18312: sigmoid

### PR DESCRIPTION
**Pull request by peterwisu**
Close #18312

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py [hunk 1]
```diff
@@ -540,3 +540,29 @@ def test_paddle_leaky_relu(
         negative_slope=0.01,
         x=x[0],
     )
+
+
+# sigmoid
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.sigmoid",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py:549]:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @peterwisu ! Thanks for opening this PR. Requested a minor change and after that, we are good to merge. Feel free to request another review when you are done with the changes.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py (hunk 1 above)
Comment (peterwisu) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py:549]: sure will do it.

**Comment by peterwisu** [comment]
@hello-fri-end Hi thank you for the code review, I have modified a code as you suggest. Thank you

**Comment by hello-fri-end** [comment]
> @hello-fri-end Hi thank you for the code review, I have modified a code as you suggest. Thank you

Hey @peterwisu ! Thanks for making the changes. Could you please resolve the merge conflicts?

**Comment by peterwisu** [comment]
Close this PR (cannot reopen PR) new PR for this tasks https://github.com/unifyai/ivy/pull/19394

**Pull request closed without merging**